src/db/Timesheet.ts

```diff
@@ -1,12 +1,59 @@
 import { z } from "zod";
 
+/**
+ * Represents the database schema for a comment
+ */
+export const CommentSchema = z.object({
+  Type: z.string(),
+  AuthorUUID: z.string(),
+  Timestamp: z.number(),
+  Content: z.string(),
+  State: z.string(),
+})
+
+/**
+ * Represents the database schema for a schedule shift entry, made by a supervisor or admin
+ */
+export const ScheduleEntrySchema = z.object({
+  StartDate: z.number(),
+  EndDate: z.number(),
+})
+
+/**
+ * Represents the database schema for a clockin/clockout pair in epoch
+ */
+export const TimeEntrySchema = z.object({
+  StartDate: z.number(),
+  EndDate: z.number(),
+})
+
+export const StatusSchema = z.object({
```

Comment:
Schema-level comment? Also, can you leave a comment to explain the possible status types?